MAUI/ImageEditor/Crop.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: Crop in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about crop support in Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Crop in .NET MAUI ImageEditor (SfImageEditor)
+
+Using the Image Editor's cropping tool, you can select and crop a particular section of an image.
+
+## Image cropping ratio
+
+The Image Editor control provides a range of selection choices for cropping images, such as custom, square, circle, and different aspect ratios. To crop a specific area of an image, simply drag and resize the selection region.
+
+There are two methods to perform the cropping operation:
+
+* Enable cropping and visually choose the cropping area.
+* Input the cropping region manually.
+
+## Handling the cropping tool
+
+The `Crop` method in the image editor control crops the image based on the `ImageCropType`. It allows users to enable or disable the cropping region displayed over the image, making it easier to visually choose the area for cropping.
+
+    * ImageCropType - Specifies the selection type for cropping the image.
+
+* The following code shows cropping the image to any desired size.
+
+{% highlight C# %}
+
+imageEditor.Crop(ImageCropType.Free);
+
+{% endhighlight %}
+
+After the cropping area has been selected, the `SaveEdits` method is called, which in turn crops the selected region and displays the cropped image on the image editor.
+
+{% highlight C# %}
+
+imageEditor.SaveEdits();
+
+{% endhighlight %}
+
+After selecting the cropping area, if you decide to cancel the edits and revert back to the original image, you can use the `CancelEdits` method. It discard the changes made during the cropping process and restore the image to its initial state.
```

Comment:
After selecting the cropping area, if you decide to cancel the edits and revert back to the original image, you can use the `CancelEdits` method. It discards the changes made during the cropping process and restores the image to its initial state.